CreateCardForStandalonePR/CreateCardForStandalonePR.ts

```diff
@@ -0,0 +1,18 @@
+import { WebhookPayload } from "@actions/github/lib/interfaces";
+import { OctokitAction } from "../lib/OctokitAction";
+
+class CreateCardForStandalonePR extends OctokitAction {
+    protected async execute(): Promise<void> {
+        const pr = this.payload.pull_request as WebhookPayload["pull_request"] & { id: number };
+        const matches = pr.body == null ? null : pr.body.match(/(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved)\s*#\d+/gi);
```

Comment:
you can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) like:

```javascript
const matches = pr.body?.match(/(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved)\s*#\d+/gi);
```